script.js

```diff
@@ -20,10 +20,16 @@ function calculateTip() {
     //Display Total
 
     document.getElementById("billTotal").innerHTML = Total;
+    //calculate tip per person
```

Comment:
Try to put comments before the specific line of code that's performing the action